src/metabase/pulse/render/body.clj

```diff
@@ -679,17 +677,25 @@
         enforced-type   (if (= chart-type :combo)
                           nil
                           chart-type)
+        card-name       (:name card)
         ;; NB: There's a hardcoded limit of arity 2 on x-axis, so there's only the 1-axis or 2-axis case
-        series          (if (= (count x-cols) 1)
-                          (single-x-axis-combo-series enforced-type joined-rows x-cols y-cols viz-settings)
-                          (double-x-axis-combo-series enforced-type joined-rows x-cols y-cols viz-settings))
+        series          [(if (= (count x-cols) 1)
```

Comment:
I should probably name this `series-seqs` to make it looks the same as other symbols in this file and its test file.

Reply:
Good callout. I note this to myself but forgot to work on it apparently...